### ISSUE TITLE
I'm confused about timestamp

### ISSUE THREAD

**Issue by wshwhyu** [ahlashkari/CICFlowMeter#69]
I use the dataset of CICDDoS2019,but There is a problem that bothers me.
The Official website DDoS Evaluation Dataset (CICDDoS2019) is https://www.unb.ca/cic/datasets/ddos-2019.html. In the site ,The author mentions that the capturing period for the training day on January 12th started at 10:30 and ended at 17:15,but I download the pcaps from the site, Open it with wireshark. Timestamp does not correspond. you can see in picture as below:
![image](https://user-images.githubusercontent.com/16811625/76416084-291a3f00-6379-11ea-8473-b2c7df74e69e.png)

The wireshark time setting is as bolow:
![image](https://user-images.githubusercontent.com/16811625/76416154-4bac5800-6379-11ea-93f0-9d0fbf3afe94.png)

 January 12th  or December 1st，I'm confused.

**Comment by Jumabek** [comment]
I dont know an answer.
However, I would not bother by the date. Instead just use the schedules in hour, min and sec.
As Prof. Habibi says, CIC-IDS-2018 attack schedules are given in AST, I suppose the same for DoS2019.

**Comment by wshwhyu** [comment]
@Jumabek thanks. How to use the CICDDoS2019 or CICDatasetCIC-IDS-2017  data set to divide the test set and training set. Because there are too many data, how to effectively and quickly select the smaller test set and training set

**Comment by ahlashkari** [comment]
Hi @wshwhyu ,
As I mentioned on the previous issue, pls ask your questions about datasets through email. I will answer them as I am answering several related questions every day also.  (BTW, @Jumabek highlighted the right point that you need to follow for these two datasets).

**Comment by jakubvsetecka** [comment]
I downloaded the pcaps from 3rd of November. I assume that is the first day, which according to https://www.unb.ca/cic/datasets/ddos-2019.html should start at 9:43 but it starts at 13:18.
![image](https://github.com/ahlashkari/CICFlowMeter/assets/98713612/e0a60c92-14ed-4199-8704-1c89156469c6)
Now when I read @ahlashkari message I looked up the last pcap file from that day and found, that the last packet was sent at 21:36.
![image](https://github.com/ahlashkari/CICFlowMeter/assets/98713612/7952944f-cf04-4593-b6aa-f29fb722901d)
According to CIC the last attack in that day should have ended at 17:35. So the supposed duration from the first to the last attack should be 17:35 - 9:43 = 7:52. However, the packets included in the pcaps from that day were being captured for 21:36 - 13:18 = 8:18.
Please let me know if I am missing something, or how am I supposed to know when an attack starts and ends.
